src/optics.jl

```diff
@@ -465,6 +465,12 @@ end
     delete(obj, IndexLens(lens.f(obj)))
 end
 
+
+Broadcast.broadcastable(
+    o::Union{PropertyLens,IndexLens,DynamicIndexLens,Elements,Properties,If,Recursive}
+) = Ref(o)
```

Comment:
```suggestion
) = Ref(o)
```
I wonder if `Ref` might allocate sometimes? Maybe `(o,)` is better?